docs/guides/08-debugging/application.md

```diff
@@ -40,19 +40,59 @@ error: aborting due to previous error
 For more information about this error, try `rustc --explain E0425`.
 ```
 
-## WebView JS Console
+## WebView Console
 
 Right-click in the WebView, and choose `Inspect Element`. This opens up a web-inspector similar to the Chrome or Firefox dev tools you are used to.
+You can also use the `Ctrl+Shift+i` shortcut on Linux and Windows, and `Command+Option+i` on macOS to open the inspector.
 
-## Create a Debug Build
+### Opening the devtools programatically
 
-There are cases where you might need to inspect the JS console in the final bundle, so Tauri provides a simple command to create a debugging bundle:
+You can control the inspector window visibility by using the `Window::open_devtools` and `Window::close_devtools` functions:
+
+```rust
+use tauri::Manager;
+tauri::Builder::default()
+  .setup(|app| {
+    #[cfg(debug_assertions)] // only include this code on debug builds
+    {
+      let window = app.get_window("main").unwrap();
+      window.open_devtools();
+      window.close_devtools();
+    }
+    Ok(())
+  });
+```
+
+### Using the inspector in production
+
+By default the inspector is only enabled in development and debug builds, unless you enable it with a Cargo feature.
+
+- Create a **debug** build
+
+To create a debug build, run the `tauri build --debug` command.
 
 <Command name="build --debug" />
 
 Like the normal build and dev processes, building takes some time the first time you run this command but is significantly faster on subsequent runs.
 The final bundled app has the development console enabled and is placed in `src-tauri/target/debug/bundle`.
 
-## Run Your App From the Terminal
-
 You can also run a built app from the terminal, giving you the Rust compiler notes (in case of errors) or your `println` messages. Just find the file `src-tauri/target/(release|debug)/[app name]` and either double click it (but be warned, the terminal closes on errors) or just run it in directly in your console.
```

Comment:
```suggestion
You can also run a built app from the terminal, giving you the Rust compiler notes (in case of errors) or your `println` messages. Browse to the file `src-tauri/target/(release|debug)/[app name]` and either double-click it (note: the terminal closes on errors) or just run it in directly in your console.
```